packages/mandb.rb

```diff
@@ -62,10 +62,10 @@ def self.install
   end
 
   def self.postinstall
-    puts 'Started mandb cache rebuild. (Errors from this can either be ignored or reported upstream to the relevant package maintainers.)'.yellow
+    puts 'Starting backgrounded mandb cache rebuild. This will be logged to /tmp/man-db-rebuild.log.'.yellow
```

Comment:
Log to `/usr/local/var/log` instead?  The `/tmp` location is not a "best practice" location for log files.

Reply:
Sure.